### PR TITLE
Add migration for priority scale

### LINKED ISSUE #4823: Add a P0-P5 scale value in poker
​

AC

- [ ] migration added for P value scale

​

Estimate: 2 hours

### PR DESCRIPTION
**Pull request by tianrunhe**
Resolves #4823 

Screen Shot 2021-03-30 at 4 51 26 PM

Screen Shot 2021-03-30 at 4 54 07 PM

### CODE REVIEW

**Comment by mattkrick** [comment]
i think you're using the schema from an older migration :wink:

here's what the data in the DB looks like

```js
{
  createdAt: {$reql_type$: 'TIME', epoch_time: 1616630539.728, timezone: '+00:00'},
  id: 'tshirtSizeScale',
  isStarter: true,
  name: 'T-Shirt Sizes',
  sortOrder: 1,
  teamId: 'aGhostTeam',
  updatedAt: {$reql_type$: 'TIME', epoch_time: 1616630539.728, timezone: '+00:00'},
  values: [
    {color: '#5CA0E5', label: 'XS'},
    {color: '#5CA0E5', label: 'SM'},
    {color: '#45E595', label: 'M'},
    {color: '#E59545', label: 'L'},
    {color: '#E59545', label: 'XL'},
    {color: '#E55CA0', label: '?'},
    {color: '#AC72E5', label: 'Pass'}
  ]
}
```

**Comment by tianrunhe** [comment]
> i think you're using the schema from an older migration 😉
> 

@mattkrick Duh! You are right! Fixed.

File: packages/server/database/migrations/20210330163028-addPriorityScale.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import {R} from 'rethinkdb-ts'
+import {PokerCards} from 'parabol-client/types/constEnums'
+import {PALETTE} from 'parabol-client/styles/paletteV3'
+
+const MAX_32_BIT_INTEGER = Math.pow(2, 31) - 1
```
Comment (mattkrick) [inline packages/server/database/migrations/20210330163028-addPriorityScale.ts:5]: nit: no longer needed (surprised typescript didn't yell about this!)

File: packages/server/database/migrations/20210330163028-addPriorityScale.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+const priorityScaleId = 'priorityScale'
+const priorityScale = [
+  {
+    id: priorityScaleId,
+    createdAt: new Date(),
+    updatedAt: new Date(),
+    name: 'Priorities',
+    isStarter: true,
+    sortOrder: 0,
```
Comment (mattkrick) [inline packages/server/database/migrations/20210330163028-addPriorityScale.ts:13]: this should be 3 so it doesn't conflict with fibonacci

Comment (tianrunhe) [reply packages/server/database/migrations/20210330163028-addPriorityScale.ts]: 🦅👀!

**Pull request merged**